### PR TITLE
Add Psi4 1.3.1 unit tests

### PR DESCRIPTION
**Pull request by berquist**

### CODE REVIEW

**Comment by langner** [comment]
Something's still failing.

Also, what do you mean by "1.2.1 unit tests weren't enabled"?

**Comment by berquist** [comment]
They were present but never actually tested for some reason :scream:

File: test/testdata [hunk 1]
```diff
@@ -52,7 +54,7 @@ Basis     Molpro      MolproBigBasisTest    basicMolpro2006       C_bigbasis.out
 Basis     Molpro      MolproBigBasisTest    basicMolpro2012       C_bigbasis.out
 Basis     NWChem      GenericBigBasisTest   basicNWChem6.0        C_bigbasis.out
 Basis     NWChem      GenericBigBasisTest   basicNWChem6.5        C_bigbasis.out
-Basis     Psi4        Psi4BigBasisTest      basicPsi4-1.0         C_bigbasis.out
+Basis     Psi4        Psi4BigBasisTest      basicPsi4-1.2.1       C_bigbasis.out
```
Comment (langner) [inline test/testdata]: Did you want to leave the 1.0 unit test around for now as well?

**Comment by berquist** [comment]
> Why didn't you add the 1.3 tests to testdata?

Because I completely forgot.

> Did you want to leave the 1.0 unit test around for now as well?

No, it's quite old now and most people are going to be using quite new versions (>= 1.3), plus the "2 latest versions as unit tests" rule is good.

**Pull request merged**